app/src/main/java/kr/co/connect/boostcamp/livewhere/data/SharedPreferenceStorage.kt

```diff
@@ -0,0 +1,38 @@
+package kr.co.connect.boostcamp.livewhere.data
+
+import android.content.Context
+import android.content.SharedPreferences
+import androidx.core.content.edit
+import kotlin.properties.ReadWriteProperty
+import kotlin.reflect.KProperty
+
+interface SharedPreference {
+    var uuid: String?
```

Comment:
```suggestion
    val uuid: String?
```
P3. var 대신 val로 선언하여도 var로 오버라이딩할 수 있습니다